<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Flipkart-MacBookPro-18.local" tests="1" failures="1" timestamp="19 Jun 2013 04:02:09 GMT" time="0.207" errors="0">
  <testcase name="testRequestCache" time="0.185" classname="ExecutionThreadTest">
    <failure type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError
	at ExecutionThreadTest.testRequestCache(ExecutionThreadTest.java:23)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:60)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:106)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:111)
	at org.apache.maven.surefire.booter.ProviderFactory$ClassLoaderProxy.invoke(ProviderFactory.java:103)
	at $Proxy0.invoke(Unknown Source)
	at org.apache.maven.surefire.booter.SurefireStarter.invokeProvider(SurefireStarter.java:150)
	at org.apache.maven.surefire.booter.SurefireStarter.runSuitesInProcess(SurefireStarter.java:91)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:69)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- testRequestCache -->
</testsuite>
